src/pages/EditStudent.tsx

```diff
@@ -156,13 +157,18 @@ const EditStudent = () => {
         msg={t("Please create your child profile").toString()}
       />
       <div className="content">
+
         {stage === STAGES.NAME && (
           <StudentNameBox
             studentName={studentName!}
             onValueChange={setStudentName}
             onEnterDown={isNextButtonEnabled() ? onNextButton : () => {}}
           />
```

Comment:
Fix Two className with the same id(content)

Reply:
> Fix Two className with the same id(content)

<IonPage id={"Edit-student-page"}> 

this one right?

Reply:
> No, This one
> 
> ```
> <div className="content">
> ```
> 
> I can see it 2 times

done, now see